fabric-common/lib/EventListener.js

```diff
@@ -59,26 +52,24 @@ class EventListener {
 	 * will be automatically unregistered.
 	 */
 
-	/*
+	/**
```

Comment:
I think this might have been left as a non-JSDoc comment so it wasn't included in the JSDoc. I don't think this constructor is intended to be called by client code. Instances are obtained only as return values from EventService.registerXXXListener() functions.

If I am correct about the intent, perhaps we could have it as a JSDoc comment but include the `@private` tag?

Reply:
add a private tag is a good option.